src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -71,14 +72,27 @@ private TexlParser(IReadOnlyList<Token> tokens, Flags flags, Features features =
             _flagsMode = new Stack<Flags>();
             _flagsMode.Push(flags);
             _features = features;
+            _parserOptions = parserOptions;
+        }
+
+        public static ParseUDFsResult ParseUDFsScript(string script, CultureInfo loc = null, bool numberIsFloat = false)
+        {
+            Contracts.AssertValue(script);
+            Contracts.AssertValueOrNull(loc);
+
+            // UDFs always support ReservedKeywords, no back compat concern
+            var formulaTokens = TokenizeScript(script, loc, Flags.NamedFormulas | (numberIsFloat ? Flags.NumberIsFloat : 0));
+            var parser = new TexlParser(formulaTokens, Flags.NamedFormulas | (numberIsFloat ? Flags.NumberIsFloat : 0), new ParserOptions());
+
+            return parser.ParseUDFs(script);

```

Comment:
Why this change?

Reply:
Existing API that was removed by Anderson recently, I kept it by mistake